examples/run/cpu/CMakeLists.txt

```diff
@@ -1,23 +1,23 @@
 # TRACCC library, part of the ACTS project (R&D line)
 #
-# (c) 2021-2022 CERN for the benefit of the ACTS project
+# (c) 2021-2023 CERN for the benefit of the ACTS project
 #
 # Mozilla Public License Version 2.0
 
 traccc_add_executable( seeding_example "seeding_example.cpp"
-   LINK_LIBRARIES vecmem::core traccc::core traccc::io 
+   LINK_LIBRARIES vecmem::core traccc::core traccc::io
    traccc::performance traccc::options detray::utils detray::io)
 
 traccc_add_executable( seq_example "seq_example.cpp"
-   LINK_LIBRARIES vecmem::core traccc::core traccc::io 
-   traccc::performance traccc::options)
+   LINK_LIBRARIES vecmem::core traccc::core traccc::io
+   traccc::performance traccc::options detray::utils detray::io)
```

Comment:
What do you need detray utils for?

Reply:
Unfortunately the build setup is pretty imperfect here... 😦 Ideally here we'd be using one of the "algebra specific" Detray targets. Like `detray::core_array`, and friends.

https://github.com/acts-project/detray/blob/main/core/CMakeLists.txt#L49-L58

Since I didn't want to bother with fixing the build up properly right now, I just took the library list from the `seeding_example` executable, which is already using `detray::detector`. I knew that this was not a clean setup, but since the build of that executable has worked since a while, I was willing to put up with the imperfection for now. 🤔

Reply:
Ok, fine for now